integration/base/src/main/java/com/inrupt/client/integration/base/MockAccessGrantServer.java

```diff
@@ -71,6 +77,40 @@ private void setupMocks() {
                     .willReturn(aResponse()
                         .withStatus(Utils.NO_CONTENT)));
 
+        wireMockServer.stubFor(post(urlEqualTo("/derive"))
```

Comment:
It might be a more realistic test if this stub also checks that the request body contains the requested resource URL. Other invocations (using the priority mechanism) should return an empty list of access grants. That, at least, would address the issue where four access grants are returned in the tests